File: aspnet-core/src/TicketTracker.Application/Tickets/Dto/UpdateTicketInput.cs
using Abp.Application.Services.Dto;
using Abp.AutoMapper;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TicketTracker.Entities;

namespace TicketTracker.Tickets.Dto { 
    [AutoMap(typeof(Ticket))]
    public class UpdateTicketInput : EntityDto<int> {
        [Required]
        public string Title { get; set; }
        public string Description { get; set; }
        public TicketPriority Priority { get; set; }
        public TicketType Type { get; set; }

        public int ComponentId { get; set; }
        public int StatusId { get; set; }
        public int? ActivityId { get; set; }
    }
}

File: reactjs/src/services/ticket/dto/updateTicketInput.ts
import { EntityDto } from "../../dto/entityDto";
import {TicketPriority} from './ticketPriority';
import {TicketType} from './ticketType';

export class UpdateTicketInput extends EntityDto { 
    title!: string; 
    description?: string; 
    priority!: TicketPriority;
    type!: TicketType;

    componentId!: number;
    statusId!: number;
    activityId?: number; 
}